addons/sms/data/neutralize.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE ir_config_parameter
+    SET value = 'https://iap-services-test.odoo.com'
+WHERE key = 'sms.endpoint';
```

Comment:
Same as `partner_autocomplete`, UPSERT to force test endpoint.
This should be done for all the IAP endpoints.

Reply:
applying the fix on all similar queries